index.js

```diff
@@ -1,16 +1,42 @@
 import './shim';
-import {AppRegistry} from 'react-native';
+import {AppRegistry, Alert} from 'react-native';
 import Root from './src/Root';
 import React from 'react';
 import './i18n';
+import { setJSExceptionHandler, setNativeExceptionHandler } from 'react-native-exception-handler';
 import {name as appName} from './app.json';
 import getStore from './src/store';
 import {Provider} from 'react-redux';
 import {PersistGate} from 'redux-persist/integration/react';
 import 'react-native-url-polyfill/auto'; // https://github.com/facebook/react-native/issues/23922#issuecomment-648096619
 import {enableFreeze} from 'react-native-screens';
+import { Analytics } from './src/store/analytics/analytics.effects';
+import { APP_VERSION } from './src/constants/config';
+import { GIT_COMMIT_HASH } from '@env';
 enableFreeze(true);
 
+const errorHandler = (e, isFatal) => {
+  if (isFatal) {
+    Analytics.track('BitPay App - Crashed App', { build: GIT_COMMIT_HASH, version: APP_VERSION});
```

Comment:
Should be: 
```
store.dispatch(Analytics.track(...));
```

and moved after the `store` declaration.

Reply:
Do we want to track the actual error as well? Or add to persistent logs?

Reply:
Yeah, good idea. I'll take a look. Thanks!